apps/bluetooth/locales/bluetooth.en-US.properties

```diff
@@ -70,7 +70,10 @@ active-pair-passkey      = To pair with {{device}}, enter the passkey on the dev
 passive-pair              = Bluetooth pairing request
 passive-pair-confirmation = {{device}} would like to pair with this phone. To accept, make sure that both devices currently have the following PIN displayed:
 passive-pair-pincode      = {{device}} would like to pair with this phone. To accept, enter the PIN on the device.
```

Comment:
oh, I did not realize that we use javascript to compose string id so I didn't find it.